data/provider/src/main/java/tech/pegasys/artemis/api/schema/BLSPubKey.java

```diff
@@ -60,8 +61,11 @@ public String toString() {
     return bytes.toString();
   }
 
-  public static BLSPubKey fromHexString(String value) {
-    return new BLSPubKey(Bytes.fromHexString(value));
+  public static BLSPubKey fromHexString(String value) throws IllegalArgumentException {
```

Comment:
nit: Don't need to declare unchecked exceptions like `IllegalArgumentException`.